rider-fsharp/build.gradle.kts

```diff
@@ -240,19 +241,26 @@ tasks {
             files = files + listOf("$testHostName.dll", "$testHostName.pdb")
         }
 
-        files.forEach {
-            from(it) { into("${intellij.pluginName}/dotnet") }
+        val filesMapping = files.zip(
+            files.map {
+                val pathEnd = if (it.contains("Fantomas")) "/fantomas" else ""
+                "${intellij.pluginName}/dotnet$pathEnd"
```

Comment:
Could you use separate lists or additional relative path params for this? The `contains("Fantomas")` check looks too ad-hoc.